### PR TITLE
805 make dashboard links clickable

### PR DESCRIPTION
**Pull request by jonathanbobel**
## Description

The dashboard had a large clickable area that didn't look like an actual link or button that allowed the user to drill down into a Status page for their recent messages. This fix removed those areas and created one single "Details" button that is more self-explanatory and allows for the same functionality. 

Additionally, I removed the section that showed failures (and the subsequent confusing link to _just_ the failures section). The user can still click on the Details button and see the combined status page, including failures. 

Upcoming designs will have data visualizations to better show the Status - all current Delivered/Pending/Failed.

### CODE REVIEW

**Review by terrazoon — APPROVED** [review]
LGTM

File: app/templates/views/dashboard/_totals.html [hunk 1]
```diff
@@ -1,43 +1,25 @@
 <div class="ajax-block-container">
   <div class="grid-row grid-gap">
     <div id="total-sms" class="grid-col-12">
-      <span class="big-number-with-status">
-        <a class="usa-link display-flex" href="{{ url_for('.view_notifications', service_id=service_id, message_type='sms', status='sending,delivered,failed') }}">
+      <span class="big-number-with-status display-block margin-bottom-2">
+        <p>
```
Comment (heyitsmebev) [inline app/templates/views/dashboard/_totals.html:5]: this p tag could be deleted. It's creating this large gap between the number and "Messages sent"
image

**Review by heyitsmebev — CHANGES_REQUESTED** [review]
I like the use of an actual button! 

One styling observation:  It seems like the font size was carried over from the original version with the blue box. Now, without the blue box, the font appears significantly larger in comparison to the rest. I'm curious if this was intentional. If it was, just disregard this comment. If not, I think you could bring it down .5rem or so to help it look more balanced 
image

**Review by heyitsmebev — APPROVED** [review]
✅

**Review by ccostino — APPROVED** [review]
Thanks, @jonathanbobel!

**Pull request merged**